packages/bundles/package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "name": "@vaadin/bundles",
+  "version": "23.0.0-beta3",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Bundles of components and dependencies",
+  "scripts": {
+    "clean": "rm -f *.js *.js.map *.tgz",
+    "build:scan": "rollup --config=rollup.config.ts --configPlugin @rollup/plugin-typescript",
+    "build:bundle": "NODE_OPTIONS=\"--loader ts-node/esm\" webpack",
+    "build": "npm run build:scan && npm run build:bundle",
+    "test": "mocha 'src/test/**/*.ts'",
+    "prepack": "npm run clean",
+    "prepare": "npm run build",
+    "postpack": "npm run test"
+  },
+  "type": "module",
+  "main": "vaadin.js",
+  "module": "vaadin.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vaadin/web-components.git",
+    "directory": "packages/bundles"
+  },
+  "keywords": [
+    "vaadin",
+    "components",
+    "webcomponents",
+    "web-components"
+  ],
+  "author": "Vaadin Ltd",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/vaadin/platform/issues"
+  },
+  "homepage": "https://github.com/vaadin/platform#readme",
+  "devDependencies": {
+    "@polymer/iron-icon": "^3.0.1",
+    "@polymer/iron-iconset-svg": "^3.0.1",
+    "@polymer/iron-media-query": "^3.0.1",
```

Comment:
Let's remove this dependency, it's no longer used. See https://github.com/vaadin/platform/pull/2754